build-farm/platform-specific-configurations/mac.sh

```diff
@@ -81,42 +81,43 @@ fi
 echo "[WARNING] You may be asked for your su user password, attempting to switch Xcode version to ${XCODE_SWITCH_PATH}"
 sudo xcode-select --switch "${XCODE_SWITCH_PATH}"
 
-# Adopt does not have builds of OpenJDK 7, OpenJDK 8 can boot itself just fine.
-if [ "${JAVA_FEATURE_VERSION}" = "8" ]; then
-  BOOT_JDK_VERSION="${JAVA_FEATURE_VERSION}"
+# No MacOS builds available of OpenJDK 7, OpenJDK 8 can boot itself just fine.
+if [ "${JDK_BOOT_VERSION}" == "7" ]; then
+  echo "No jdk7 boot JDK available on MacOS using jdk8"
+  MAC_BOOT_JDK_VERSION="8"
```

Comment:
Can we just set `JDK_BOOT_VERSION=8` here and use that through the script to be consistent with the other platforms?

Reply:
(Obviously since you have two approvals without me you could merge it anyway, but it would be nice to change this ;-) )

Reply:
we could, my thinking was that was changing the upstream script value, say other scripts were being called that... but i'm just following a not changing the caller's value principle when it's actually not relevant here anyway...

Reply:
nw, changed